doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -84,6 +84,18 @@ nRF Desktop
   * Updated documentation with information about forwarding boot reports.
     See the documenation page of nRF Desktop's :ref:`nrf_desktop_hid_forward` for details.
   * Fixed an issue that was causing the HID keyboard LEDs to remain turned on after host disconnection while no other hosts were connected.
+  * The application configuration files were switched to follow a generic scheme.
+    When selecting the build file, instead of setting up a CMAKE_BUILD_TYPE, point to build type specific prj file using CONF_FILE.
+    E.g. CONF_FILE=prj_release.conf will now be used instead of CMAKE_BUILD_TYPE=ZRelease.
```

Comment:
The configuration steps in nRF Desktop's README.rst are not updated with this information.

Reply:
Can you elaborate? Should I add
`Documentation updated accordingly.` or `The board configuration steps in nrf_desktop documentation was updated`.
Or I should add something more to the README file?

Reply:
I mean that we're describing how to select the build file in the changelog, but not in the actual documentation. We discussed this on call later that day though, so I understand you have a documentation task to add this information to nRF Desktop README.